test.sh

```diff
@@ -0,0 +1,13 @@
+curl \
```

Comment:
I am ok with having this, but we should iterate on it.

Let's have a python test suite that will start the server + db and do some tests.

This test counts with a database state which is only on your machine.

In the end this should be run by a Github action

Reply:
Definitely,  this is just a sample with some quick tests. Seemed better to leave it for others to just easily adjust it and run, than to figure it out from scratch.